darwin/Classes/AudioPlayer.m

```diff
@@ -1105,7 +1105,11 @@ - (void)seek:(CMTime)position index:(NSNumber *)newIndex completionHandler:(void
                 // If playing, buffering will be detected either by:
                 // 1. checkForDiscontinuity
                 // 2. timeControlStatus
-                [_player play];
+                if (@available(iOS 10.0, *)) {
+                    [_player playImmediatelyAtRate:_speed];
+                } else {
+                    [_player play];
```

Comment:
If it fails to resume playback at the correct speed, for this else case how about we do this?

```dart
    _player.rate = _speed;
```

Reply:
yeah that's a good point. That should work.
Sry since most devices have iOS>10, i forgot that bit.

Reply:
done!